app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntryEffectHandler.kt

```diff
@@ -98,7 +98,7 @@ class CustomPrescriptionEntryEffectHandler @AssistedInject constructor(
     return ObservableTransformer { effects ->
       effects
           .observeOn(io)
-          .map { prescriptionRepository.prescriptionImmediate(it.prescriptionUuid) }
+          .map { prescriptionRepository.prescription(it.prescriptionUuid) }
```

Comment:
This will break the remove prescription flow since we are implicitly depending on the reactive change notification to detect if a prescription has been deleted.

Reply:
Fixed this by retaining and using the reactive version of this method.